napari/components/viewer_model.py

```diff
@@ -341,6 +346,11 @@ def _new_labels(self):
         empty_labels = np.zeros(shape, dtype=int)
         self.add_labels(empty_labels, translate=np.array(corner), scale=scale)
 
+    def _slice_layer(self, event: Event) -> None:
```

Comment:
Consider renaming to `_on_layer_reslice` for consistency with other callbacks in the codebase.

Reply:
Done. Pushing is pending.